commit 42add0132dad8c32ec356d59d0f7daceb9d42ed8
Author: Sven Wegener <sven.wegener@stealer.net>
Date:   Tue Jul 22 02:18:03 2008 +0200

    Fix configure.ac libotr error
    
    This fixes the following error:
    
    checking for otrl_init in -lotr... yes
    checking libotr/context.h usability... yes
    checking libotr/context.h presence... yes
    checking for libotr/context.h... yes
    ./configure: line 15006: libotr/userstate.h: No such file or directory
    
    Signed-off-by: Sven Wegener <sven.wegener@stealer.net>
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 04ca231..56cf974 100644
--- a/configure.ac
+++ b/configure.ac
@@ -261,7 +261,7 @@ if test "x$with_libotr" != "xno"; then
 
 	AC_CHECK_LIB(otr, otrl_init,
 	[
-		AC_CHECK_HEADERS(libotr/context.h, libotr/userstate.h
+		AC_CHECK_HEADERS(libotr/context.h libotr/userstate.h,
 		[
 			AC_DEFINE(HAVE_LIBOTR, 1, [define if you have libotr])
             OTR_LIBS="-lotr $LIBS"
